File: MXApp/src/main/java/info/mx/tracks/recyclerview/MyListItem.java
package info.mx.tracks.recyclerview;

import android.database.Cursor;

public class MyListItem {
    private String name;

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public static MyListItem fromCursor(Cursor cursor) {
        //TODO return your MyListItem from cursor.
        return null;
    }
}

File: app/src/main/java/info/mx/tracks/recyclerview/MyListItem.kt
package info.mx.tracks.recyclerview

import android.database.Cursor

class MyListItem {
    var name: String? = null

    companion object {
        fun fromCursor(cursor: Cursor?): MyListItem? {
            //TODO return your MyListItem from cursor.
            return null
        }
    }
}
